.gitignore

```diff
@@ -56,10 +60,4 @@ sysinfo.txt
 *.unitypackage
 
 # Crashlytics generated file
-crashlytics-build.properties
-
-# Rider settings
-.idea/
-
-# Unity layout files
-UserSettings/Layouts/
\ No newline at end of file
+crashlytics-build.properties
```

Comment:
Why changed the git ignore?

Reply:
Wanted to see if that helps ignore the UserSettings folder, what I did is just move /UserSettings/ and .idea/ statement to the front